integreat_cms/static/src/js/pois/poi-map.ts

```diff
@@ -0,0 +1,51 @@
+import maplibregl from "maplibre-gl";
+
+window.addEventListener("load", () => {
+  let longitude = document.getElementById("id_longitude") as HTMLInputElement;
+  let latitude = document.getElementById("id_latitude") as HTMLInputElement;
+
+  let map = document.getElementById("map");
+
+  showPreview(map);
+
+  longitude.addEventListener("focusout", ()=> showPreview(map));
+  latitude.addEventListener("focusout",  ()=> showPreview(map));
+
+  longitude.addEventListener("input", ()=> showPreview(map));
+  latitude.addEventListener("input", ()=> showPreview(map));
+
+  document.getElementById("auto-fill-coordinates")?.addEventListener("input", ({ target }) => {
+    if (!(target as HTMLInputElement).checked) {
+      showPreview(map);
+    }
+  });
+});
+
+
+function showPreview(container:HTMLElement){
```

Comment:
```suggestion
function showPreview(){
  let container = document.getElementById("map");
```
Since we only have one map element now, I guess we can remove this function argument.